### PR TITLE
Create Howard_Parry.yaml

### PR DESCRIPTION
**Pull request by showmic09**

### CODE REVIEW

**Comment by mwestphall** [comment]
@opensciencegrid/project-office do we have permission to add the new institution "Howard University" to the institutions database?

**Comment by jstathas** [comment]
Approved! And this is awesome to add Howard.

Janet

From: mwestphall ***@***.***>
Date: Friday, August 9, 2024 at 11:07 AM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] Create Howard_Parry.yaml (PR #4067)

@opensciencegrid/project-office<https://github.com/orgs/opensciencegrid/teams/project-office> do we have permission to add the new institution "Howard University" to the institutions database?

File: projects/Howard_Parry.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Department: Microbiology
+Description: The thrust of this project is to obtain a functional, structural and
+  mechanistic understanding of iron transport and systemwide homeostasis.
+FieldOfScience: Biophysics
+FieldOfScienceID: '26.0203'
+InstitutionID: Unknown
```
Comment (mwestphall) [inline projects/Howard_Parry.yaml:6]:
```suggestion
InstitutionID: https://osg-htc.org/iid/6flk2zw30csu
```

**Review by mwestphall — APPROVED** [review]

**Comment by jstathas** [comment]
https://ror.org/05gt1vc06


From: mwestphall ***@***.***>
Date: Friday, August 9, 2024 at 2:13 PM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] Create Howard_Parry.yaml (PR #4067)

@mwestphall commented on this pull request.

________________________________

In projects/Howard_Parry.yaml<https://github.com/opensciencegrid/topology/pull/4067#discussion_r1712021365>:

> @@ -0,0 +1,8 @@

+Department: Microbiology

+Description: The thrust of this project is to obtain a functional, structural and

+  mechanistic understanding of iron transport and systemwide homeostasis.

+FieldOfScience: Biophysics

+FieldOfScienceID: '26.0203'

+InstitutionID: Unknown
⬇️ Suggested change

-InstitutionID: Unknown

+InstitutionID: https://osg-htc.org/iid/6flk2zw30csu

**Pull request merged**